packages/web-components/src/components/rux-global-status-bar/rux-global-status-bar.scss

```diff
@@ -112,10 +122,9 @@ slot[name='left-side'] > .shifted-up,
     font-size: var(--font-body-2-font-size);
     font-weight: var(--font-body-2-font-weight);
     letter-spacing: var(--font-body-2-letter-spacing);
-    padding: 2px 6px;
+    padding: calc(var(--spacing-050) + var(--spacing-025)) var(--spacing-2); //3px 8px
     text-align: center;
-    height: 22px;
-    margin-right: 6px;
+    margin-right: var(--spacing-2); //6px
```

Comment:
**NIT**: When the root font size is `16px`, the `--spacing-2` variable ends up resolving to `8px`. The `6px` note here may be confusing.